caddytest/integration/caddyfile_adapt/not_block_with_named_matcher.txt

```diff
@@ -0,0 +1,107 @@
+:80
+
+@matcher {
+    header Abc "123"
+    header Bcd "456"
+}
+respond @matcher "Abc Bcd headers matched"
+
+@not-matcher {
+	not @matcher
+}
+respond @not-matcher 403
+
+@nested-not {
+	not {
+		not @matcher
+	}
+}
+respond @nested-not 401
```

Comment:
Single line syntax should also work; you can add a test like this as well:

```
@nested-not-single-line not not @matcher
respond @nested-not-single-line 401
```

Reply:
Added this test.